web/models.py

```
@@ -3046,3 +3046,85 @@ class Response(models.Model):
+            'username': self.user.username,
+            'full_name': f"{self.user.first_name} {self.user.last_name}",
+            'joined_at': self.joined_at.isoformat(),
+            'seat_id': self.seat_id,
+            'last_active': self.last_active.isoformat()
+        }
+
+class VirtualClassroomWhiteboard(models.Model):
+    """Model to store whiteboard data for each virtual classroom"""
+    classroom = models.OneToOneField(VirtualClassroom, on_delete=models.CASCADE, related_name='whiteboard')
+    canvas_data = models.JSONField(default=dict, blank=True)
+    background_image = models.TextField(blank=True, null=True)
```

Ruff (0.11.9)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)